tests/Progress/SnapshotTest.php

```diff
@@ -20,7 +22,7 @@ public function of_activity_instance_returns_no_modules_if_no_modules_created()
     {
         $activity = factory(Activity::class)->create();
         $activityInstance = factory(ActivityInstance::class)->create(['activity_id' => $activity->id]);
-        $progress = (new Snapshot())->ofActivityInstance($activityInstance);
+        $progress = (new Snapshot(new ProgressUpdateRepository()))->ofActivityInstance($activityInstance, 'called_id');
```

Comment:
We're testing the snapshot class, not the progress update repository class, so we're not allowed to use `new ProgressUpdateRepository`. SHould mock that class! Replace this line with the following, tested and it works:

```
        $progress = (new Snapshot($this->prophesize(\BristolSU\Support\Progress\Contracts\ProgressUpdateContract::class)->reveal()))->ofActivityInstance($activityInstance, 'called_id');
```

This will apply to all the tests in this class that were initially here